MaxText/llama_or_mistral_ckpt.py

```diff
@@ -129,29 +192,25 @@ def convert(base_model_path, maxtext_model_path, model_size):
   base_num_kv_heads = model_params["num_kv_heads"]
   vocab_size = model_params["vocab"]
   num_experts = model_params["num_experts"] if "num_experts" in model_params else None
+  mem_info = psutil.Process()
+  logging.debug("Memory usage: %f GB", mem_info.memory_info().rss / (1024 ** 3))
 
-  print(f"Loading the base model from {base_model_path}")
+  max_logging.log(f"Loading the base model from {base_model_path}")
   # Skip any hidden files for checkpoints
   ckpt_paths = sorted(pathlib.Path(base_model_path).glob("[!.]*.pth"))
   pytorch_vars = {}
   for i, ckpt_path in enumerate(ckpt_paths):
-    print(f"Loading checkpoint {i+1} of {len(ckpt_paths)} ...")
+    max_logging.log(f"Loading checkpoint {i+1} of {len(ckpt_paths)} ...")
     checkpoint = torch.load(ckpt_path, map_location="cpu")
     pytorch_vars[int(ckpt_path.name.split(".", maxsplit=2)[1])] = checkpoint
   pytorch_vars = [pytorch_vars[i] for i in sorted(list(pytorch_vars.keys()))]
+  # map weight names if they use HuggingFace instead of PyTorch convention
```

Comment:
Will this apply to existing ckpt conversion? Could you help run 2 tests to existing model to ensure no breakage?

1) llama2-7b conversion still works ([example]https://github.com/google/maxtext/blob/da50760ac0baf3920305a365215f6f0c5f110ad2/end_to_end/tpu/llama2/7b/test_llama2_7b.sh))
2) 8x7b model size still works ([example](https://github.com/google/maxtext/blob/da50760ac0baf3920305a365215f6f0c5f110ad2/end_to_end/tpu/mixtral/8x7b/1_test_mixtral.sh))

Thank you!